code/src/Album.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export const Album = (props) => {
+    console.log(props)
+    
+ return  <div className="albumCard">
+    <img src={props.album.images[1].url} alt=""/>
```

Comment:
here you are selecting the second image in the array, is that on purpose?

Reply:
Yes, there was an option of three different sizes and I chose the middle one.